src/uiScreens.c

```diff
@@ -215,3 +220,152 @@ void ui_displayNetworkParamsScreen(
 	        callback
 	);
 }
+
+void ui_displayHexBufferScreen(
```

Comment:
I'm wondering, why can't we just reuse the logic to display the pool id from stakepool delegation ui (signTx_handleCertificate_ui_runStep)?

Reply:
Well, I don't see an elegant way to do it. Manipulation with strings is quite clumsy in C. It would be nice if I could display pool id for stake delegation certificate in the same place in code as for other certificates, but it only arrives in the following APDU in the sub-machine (I can send it in the certificate one, but then I have to store it, avoid having two different copies in two places (I definitely want to avoid passing pool params to the tx hash builder separately), and the approach would not be very consistent --- showing some parameters in the main machine, some in the sub-machine).